tcms/testplans/templates/testplans/mutable.html

```diff
@@ -24,6 +32,18 @@
                 </div>
             </div>
 
+        {% if test_plan %}
+            <div class="form-group">
+                <label for="id_active" class="col-md1 col-lg-1"> {% trans "Active" %}</label>
+                <div class="col-md-1 col-lg-1"> {{ form.is_active }} </div>
```

Comment:
Try using 1 col for label, 3 cols for the input checkbox. Elsewhere we have this pattern and it works great.

Also don't use `{{ form.is_active }}` but specify the checkbox by hand, see search.html. This should not be a simple checkbox but instead a bootstrap-switch.